marquez_client/client_wo.py

```diff
@@ -196,107 +172,28 @@ def create_job_run(self, namespace_name, job_name, run_id,
         return response
 
     def mark_job_run_as_started(self, run_id):
-        log.debug("mark_job_run_as_started()")
-
         return self.__mark_job_run_as(run_id, 'start')
 
     def mark_job_run_as_completed(self, run_id):
-        log.debug("mark_job_run_as_completed()")
-
         return self.__mark_job_run_as(run_id, 'complete')
 
     def mark_job_run_as_failed(self, run_id):
-        log.debug("mark_job_run_as_failed()")
-
         return self.__mark_job_run_as(run_id, 'fail')
 
     def mark_job_run_as_aborted(self, run_id):
-        log.debug("mark_job_run_as_aborted()")
-
         return self.__mark_job_run_as(run_id, 'abort')
 
     def __mark_job_run_as(self, run_id, action):
-        log.debug("__mark_job_run_as()")
-
-        MarquezWriteOnlyClient._is_valid_uuid(run_id, 'run_id')
+        Utils.is_valid_uuid(run_id, 'run_id')
 
         return self._backend.post(
-            self._url('/jobs/runs/{0}/{1}', run_id, action),
+            self.path('/jobs/runs/{0}/{1}', run_id, action),
```

Comment:
this is where you want to add the `at` query parameter

Reply:
yup. thank you.